src/units/abilities.cpp

```diff
@@ -1188,6 +1194,15 @@ static std::optional<std::string> select_damage_type(const unit_ability_list& ab
 				return type_count[lhs] < type_count[rhs];
 			});
 		}
+		if(is_alternative){
+			if(other_){
+				std::sort( std::begin( type_list ) , std::end( type_list ) , [&]( const std::string& rhs , const std::string& lhs ){
```

Comment:
Confusing to call the left argument rhs and the right argument lhs. I'd either switch them or not call them lhs and rhs.

The comparison below should use the same order as the function arguments.

Reply:
names changed